modules/introduction/partials/new-features-80.adoc

```diff
@@ -120,6 +120,12 @@ Additional information sent by clients at connection time can be found in the lo
 https://jira.issues.couchbase.com/browse/MB-61048[MB-61048]::
 Once faulty remote cluster credentials are fixed, XDCR will now be able to more quickly restart replications that depend on the repaired references.
 
+https://jira.issues.couchbase.com/browse/MB-11575[MB-11575]::
+XDCR now supports the identification of Incoming Replications on a cluster.
+The Incoming Replications can be seen on the XDCR tab of the UI or by using the REST API command `GET /xdcr/sourceClusters`.
+For more information, see xref:rest-api:rest-xdcr-list-incoming-replications.adoc[Listing Incoming Replications].
```

Comment:
Also add the info for UI:
`From the UI, to view incoming replications information, see xref:manage:manage-xdcr/incoming-xdcr-replications.adoc#view-incoming-replications[View Incoming Replications] in xref:manage:manage-xdcr/incoming-xdcr-replications.adoc[Incoming Replications].  For information on the REST API, see xref:rest-api:rest-xdcr-list-incoming-replications.adoc[Listing Incoming Replications].`